code/modules/roguetown/roguejobs/cook/tools/pot.dm

```diff
@@ -11,17 +11,41 @@
 	w_class = WEIGHT_CLASS_BULKY
 	drop_sound = 'sound/foley/dropsound/shovel_drop.ogg'
 
+	// Sorry Melbert for butchering your code
+	var/list/obj/item/added_ingredients
+
+
+/obj/item/reagent_containers/glass/bucket/pot/attackby(obj/item/reagent_containers/food/snacks/attacking_item, mob/user, params)
+	. = ..()
+	if(.)
```

Comment:
why are you doing this if check exactly?

Reply:
Unironically? It was part of the code I was referencing and I'm too scared to ask what it's for

Reply:
okay that makes sense